Library/Homebrew/cask/download.rb

```diff
@@ -24,6 +24,8 @@ def initialize(cask, quarantine: nil)
 
     sig { override.returns(T.nilable(::URL)) }
     def url
+      return if cask.url.nil?
```

Comment:
Small thing that will give us better error messages in the future if we try downloading casks without urls.